utils/CMakeLists.txt

```diff
@@ -66,7 +66,6 @@ endif()
 if(WIN32)
    target_compile_definitions(VkExtLayer_utils PUBLIC _CRT_SECURE_NO_WARNINGS)
 endif()
-install(TARGETS VkExtLayer_utils DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
FYI @lunarpapillo this library should never have been installed. So I removed it as well.